game_remix.html

```diff
@@ -1,5 +1,5 @@
 <script type="text/javascript">
```

Comment:
If you're going to keep your JS separate from HTML, this file should be named with a '.js' extension, and you should remove the script tags.